doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -445,16 +429,23 @@ Requests to the environment
       by_notation ::= @string {? % @scope_key }
 
    Displays the full name of objects from |Coq|'s various qualified namespaces such as terms,
-   modules and Ltac.  It also displays notation definitions.
+   modules and Ltac, thereby showing the module they are defined in.  It also displays notation definitions.
 
-   If the argument is:
-
-   * :n:`@qualid` - displays the full name of objects that
-     end with :n:`@qualid`, thereby showing the module they are defined in.
-   * :n:`@string {? "%" @ident }` - displays the definition of a notation.  :n:`@string`
+   * :n:`@qualid` - refers to object names that end with :n:`@qualid`.
+   * :n:`@string {? % @scope_key }` - refers to definitions of notations.  :n:`@string`
```

Comment:
Since you are modifying these lines, what about using the description lists that we have adopted since then instead of bullet points?